.vscode/settings.json

```diff
@@ -64,7 +64,10 @@
         "apps"
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
+    "scm.diffDecorationsGutterPattern": {
+        "added": true
+    }
```

Comment:
this setting (or its removal) does nothing on my end, does this fix the diff sidebar things for you? what if you change it in your user settings instead of workspace?